spec/integration/util/windows/security_spec.rb

```diff
@@ -388,8 +387,8 @@ def grant_everyone_full_access(path)
 
         describe "#owner=" do
           it "should accept a user sid" do
-            winsec.set_owner(sids[:admin], path)
-            expect(winsec.get_owner(path)).to eq(sids[:admin])
+            winsec.set_owner(sids[:current_user], path)
+            expect(winsec.get_owner(path)).to eq(sids[:current_user])
```

Comment:
Got it; I'll look more into why we `set_owner(:guest, path)` before the tests. Maybe we can change that and have the test verify it works.